gulpfile.js

```diff
@@ -27,7 +27,8 @@ gulp.task('mocha', function () {
   return gulp
     .src(['./test/**/*', '!./test/temp'])
     .pipe(mocha({
-      reporter: 'spec'
+      reporter: 'spec',
+      timeout: Infinity
```

Comment:
@codydaig Then @ilanbiala vote is deciding, where we go to. I strongly prefer returning `this.timeout(0)` to each of the tests, thus making the tests independent of what is used to run them - _gulp_ or just clean _mocha_.